cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -969,6 +982,7 @@ func (rc *rootConfig) registerRootFlags(fs *flag.FlagSet) {
 	fs.DurationVar(&rc.mergeInterval, "merge_interval", time.Hour, "run merge this often")
 	fs.Int64Var(&rc.targetSize, "merge_target_size", getEnvWithDefaultInt64("SRC_TARGET_SIZE", 2000), "the target size of compound shards in MiB")
 	fs.Int64Var(&rc.minSize, "merge_min_size", getEnvWithDefaultInt64("SRC_MIN_SIZE", 1800), "the minimum size of a compound shard in MiB")
+	fs.Int64Var(&rc.indexConcurrency, "index_concurrency", getEnvWithDefaultInt64("SRC_INDEX_CONCURRENCY", 1), "the number of concurrent index jobs to run.")
```

Comment:
I guess we could use fs.IntVar instead and avoid the conversion later?

Reply:
I did this to get the free helper `getEnvWithDefaultInt64`. To be honest, this all needs a bit of a cleanup. Want me to tackle now or later?

Reply:
later is fine ;-)